sdk/typescript/src/types/transactions.ts

```diff
@@ -22,20 +24,53 @@ export type TransactionData = {
   gas_payment: RawObjectRef;
   gas_budget: number;
 };
-export type Transaction = {
-  data: TransactionData;
-  tx_signature: string;
-};
 
 // TODO: support u64
 export type EpochId = number;
 
-export type CertifiedTransaction = {
+export type AuthorityQuorumSignInfo = {
   epoch: EpochId;
-  transaction: Transaction;
   signatures: RawAuthoritySignInfo[];
 };
 
+export type CertifiedTransaction = {
+  data: TransactionData;
+  tx_signature: string;
+  auth_sign_info: AuthorityQuorumSignInfo;
+};
+
+export type GasCostSummary = {
```

Comment:
@lxfind what's the formula for total gas used that we display to users? Is it `computation_cost + storage_cost`?

Reply:
Good question! `computation_cost + storage_cost` is the total amount charged, but there is also `storage_rebate` rebated back. So depending on which one is bigger, they may end up either getting charged, or given money.